index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mahmoud Makhlouf</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="wraper">
+
+        <header>
+            <nav class="header">
+                <a href="#">
+                    <h1>Mahmoud Makhlouf</h1>
+                </a>
+    
+                <a href="#">
+                    <img src="Union.png" alt="collapsed-menu">
+                </a>
+            </nav>
+        </header>
+
+        <section class="container">
+
+            <div class="headline">
+                <h1>Hey There. I’m Mahmoud</h1>
+                <h3>I’am a Software Developer</h3>
```

Comment:
## OPTIONAL

- [ ] Please the paragraph (e.g., "I'm a software ...") is a `p` tag